crates/pipeline_manager/src/auth.rs

```diff
@@ -268,7 +276,15 @@ pub(crate) fn aws_auth_config() -> AuthConfiguration {
     validation.set_audience(&[audience]);
     validation.set_issuer(&[iss]);
     AuthConfiguration {
-        provider: Provider::AwsCognito(jwk_uri),
+        provider: Provider::AwsCognito(ProviderAwsCognito {
+            jwk_uri,
+            region: env::var("AWS_AMPLIFY_REGION")
```

Comment:
Are these environment variables not the same as AUTH_CLIENT_ID and AUTH_ISSUER? Why is AMPLIFY in the name?

Going by https://docs.aws.amazon.com/cognito/latest/developerguide/login-endpoint.html, all we should need us the client-id and issuer-url?

Reply:
They are - for server-side validation. Here I added the fields need for client-side configuration of auth flow; all of them are public, so should be no problem exposing them over API

Reply:
No my question is if they overlap with AUTH_CLIENT_ID and AUTH_ISSUER? If so, we should not be defining more environment variables.

Reply:
Worked out a minimal set of env variables with lalith.